airframe-sql/src/main/scala/wvlet/airframe/sql/model/Expression.scala

```diff
@@ -270,19 +270,37 @@ object Expression {
 
   case class AllColumns(
       qualifier: Option[QName],
-      sourceTables: Option[Seq[Catalog.Table]],
+      resolvedAttributes: Option[Seq[ResolvedAttribute]],
```

Comment:
This needs to be `Seq[Expression]` because the input column can be an arbitrary sub query (e.g, select * from select 1).

An idea is adding `ResolvedAllColumns(qualifier, resolvedAttributes:Seq[Expression], nodeLocation)` class.

Reply:
Hmm, besides `AllColumns`, `select 1` doesn't work in some cases:
```
-- This works
select cnt from (select 1 as cnt) => List(SingleColumn(Literal(1)))

-- But this doesn't work
select cnt from (select cnt from (select 1 as cnt)) => List(SingleColumn(Id(cnt)))
```

Reply:
> This needs to be Seq[Expression]

I think `Seq[Attribute]` is fine.